baram/iam_manager.py

```diff
@@ -61,3 +62,35 @@ def list_group_policies(self, user_group_name, max_items=100):
         :return:
         '''
         return self.cli.list_group_policies(GroupName=user_group_name, MaxItems=max_items)
+
+    def get_policies(self, scope: str = 'All', max_result: int = 1000):
+        """
+        Lists all policies in IAM
+        :param scope: 'Local' for customer managed policies, 'AWS' for AWS managed policies, 'All' for all policies
+        :param max_result: max number of results (max=1000)
+        :return:
+        """
+        policies = self.cli.list_policies(Scope=scope, MaxItems=max_result)
+        result = policies['Policies']
+        if policies['IsTruncated']:
+            marker = policies['Marker']
+            while True:
```

Comment:
Any possibility to avoid `while True`? it's close to anti design pattern to write a code.

Reply:
This was my opinion(that is why this also exists in [here](https://github.com/lks21c/baram/pull/31#discussion_r1118659890)😂). We'll remove this risk.

Reply:
I modified the code to `while policies['IsTruncated']`. In this way, it could stop the loop in more precise manner.